packages/search-ui-workplace-search-connector/bin/build-js

```diff
@@ -0,0 +1,7 @@
+# clear directory
+rm -rf ./lib
```

Comment:
can remove this line

Reply:
👍 Removed https://github.com/elastic/search-ui/pull/650/commits/e39a83a0e22911acdc8154845ead1038d2b3549e